recipe/meta.yaml

```diff
@@ -31,17 +31,17 @@ test:
     - test -f $PREFIX/include/tapi/tapi.h
 
 about:
-  home: https://opensource.apple.com/source/tapi
+  home: https://github.com/tpoechtrager/apple-libtapi
```

Comment:
yes, the fork we use includes fixes to build on linux among other changes. we could move to the apple version and apply the patches in the feedstock, but I don't think it would provide much benefit here.

Reply:
It would be better security wise to rely on the official sources I think.

Reply:
this is just a rebuild, I'm not going to change this now

Reply:
>this is just a rebuild, I'm not going to change this now

You did change ti, so I would revert that change and go back to https://opensource.apple.com/source/tapi.

Reply:
have you tried opening this url?

Reply:
Ah this is just the home. I thought you were changing the source, my bad. NVM.

Reply:
IMHO it is not very productive to add comments to pull requests without barely reviewing them.